### PR TITLE
629 backend public schools data update refactor

### PR DESCRIPTION
**Pull request by trbmcginnis**
The new data updates for public schools combines lcgms and bluebook datasets into one dataset called ceqr_school_buildings. This PR implements that changes necessary in the backend in response to these data updates.

1. **model Update**: Update the public_schools_analysis model to remove lcgms and bluebook and replace with ceqr_school_buildings. Remove private methods for lcgms that assigned subdistricts/geometry to each object in the array of objects because that is now happening in the database.
2. **Interface update**: Remove the bluebook and lcgms interfaces and add a ceqr_school_buildings interface.
3. **Resource update**: Remove bluebook and lcgms from the public_schools_analysis_resource and add ceqr_school_buildings. 
4. **Test Update**: (1) remove lcgms & bluebook interface spec and add ceqr_school_buildings interface spec and (2) update the model test public_schools_analysis_spec

### CODE REVIEW

File: backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module CeqrData
-  class ScaBluebook < Base  
-    self.schema = "sca_bluebook"
+  class CeqrSchoolBuildings < Base  
+    self.schema = "ceqr_school_buildings"
 
 		# all high schools (HS, ISHS) within project borough
 		def high_schools_in_boro(borocode)
```
Comment (bfreeds) [inline backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb]: Do we need to also update the interfaces for High Schools and Intermediate Schools?

I like the name of this function `high schools_in_boro...`, and think that all three functions in this file should follow a similar naming (`primary_schools_in_subdistricts` and `intermediate_schools_in_subdisricts`).

File: backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb [hunk 2]
```diff
@@ -13,9 +13,9 @@ def high_schools_in_boro(borocode)
 		# all primary schools (PS, PSIS) within project district and subdistrict
 		def ps_schools_in_subdistricts(subdistrict_pairs)
 			@dataset.select(
-        :geom, :district, :subdistrict, :borocode, :bldg_name, :excluded, :bldg_id, :org_id, :org_level, :name, 
-		    :address, :ps_capacity, :ps_enroll, :ms_capacity, :ms_enroll, :hs_capacity, :hs_enroll, :ogc_fid
-		  ).where(Sequel.lit("(district, subdistrict) IN (VALUES #{subdistrict_pairs.join(',')})")).where(Sequel.ilike(:org_level, '%PS%'))
+        		:geom, :district, :subdistrict, :borocode, :bldg_name, :excluded, :bldg_id, :org_id, :org_level, :name, 
+		    	:address, :ps_capacity, :ps_enroll, :ms_capacity, :ms_enroll, :hs_capacity, :hs_enroll, :ogc_fid
+		  	).where(Sequel.lit("(district, subdistrict) IN (VALUES #{subdistrict_pairs.join(',')})")).where(Sequel.ilike(:org_level, '%PS%'))
```
Comment (bfreeds) [inline backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb]: There are a couple column name changes incoming from `ceqr_school_buildings`:
- `ps_capacity` => `pc`
- `ps_enroll` => `pe`
- `ms_capacity` => `ic`
- `ms_enroll` => `ie`
- `hs_capacity` => `hc`
- `hs_enroll` => `he`

Additionally, we need to add the new `source` field, which is where we'll get `bluebook` or `lcgms`

Comment (trbmcginnis) [reply backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb]: Will add!

File: backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb (hunk 1 above)
Comment (trbmcginnis) [reply backend/app/interfaces/ceqr_data/ceqr_school_buildings.rb]: Good point

**Comment by trbmcginnis** [comment]
Closing this because it's now covered in PR #640

**Pull request closed without merging**